client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -561,13 +571,19 @@ void main(void)
 	// Note: clarity = (1 - fogginess)
 	float clarity = clamp(fogShadingParameter
 		- fogShadingParameter * length(eyeVec) / fogDistance, 0.0, 1.0);
+
+#ifdef ENABLE_TINTED_FOG
 	float fogColorMax = max(max(fogColor.r, fogColor.g), fogColor.b);
 	// Prevent zero division.
 	if (fogColorMax < 0.0000001) fogColorMax = 1.0;
 	// For high clarity (light fog) we tint the fog color.
 	// For this to not make the fog color artificially dark we need to normalize using the
 	// fog color's brightest value. We then blend our base color with this to make the fog.
 	col = mix(fogColor * pow(fogColor / fogColorMax, vec4(2.0 * clarity)), col, clarity);
+#else
+	col = mix(fogColor, col, clarity);
+#endif
```

Comment:
if I understand correctly we now have a configuration option to disable a previously added visual feature (fog tint), right?

Reply:
Yes